packages/coinstac-ui/app/render/components/consortium-card.js

```diff
@@ -25,7 +25,7 @@ export default function ConsortiumCard(props) {
 
   if (isOwner) {
     deleteButton = (
-      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium}>
+      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium} id={`delete-${label}`}>
```

Comment:
The reason I've added these `id` tags is to be able to grab them from WebdriverIO. In the e2e test, I don't have access to the `id` of a given consortium. I have the name I assign to it, though. Would you be alright with using a modified `label` with spaces replaced by dashes?